### ISSUE TITLE
sails-mongo@0.11.0 when select finished not callback

### ISSUE THREAD

**Issue by xchl** [balderdashy/sails#2906]
![Uploading ADA564C2-1870-40DC-AA30-5C76494D2984.png…]()

change version sails-mongo@0.10.6 is good.

**Comment by tjwebb** [comment]
You need to provide more information. It's not at all clear what your issue is or what you want help with.

**Comment by sailsbot** [comment]
Thanks for posting, @xchl.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
